src/components/app.js

```diff
@@ -395,6 +416,11 @@ const App = () => {
         updateConfigParameter(PARAMS_USE_NAME_KEY, useName);
     };
 
+    const handleLanguageClick = (language) => {
```

Comment:
you should also dispatch the new value to the store

Reply:
The value is dispatched to the store when the websocket notifies that the server has stored the new value. All configs behave this way. This PR should not change that so the code is correct like this I think